src/api/SFA.DAS.Provider.Events.Application/Submissions/GetLatestLearnerEventByStandardQuery/GetLatestLearnerEventByStandardQueryRequestValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.Provider.Events.Application.Validation;
+
+namespace SFA.DAS.Provider.Events.Application.Submissions.GetLatestLearnerEventByStandardQuery
+{
+    public class GetLatestLearnerEventByStandardQueryRequestValidator : IValidator<GetLatestLearnerEventByStandardQueryRequest>
+    {
+        public async Task<ValidationResult> Validate(GetLatestLearnerEventByStandardQueryRequest item)
+        {
+            var validationErrors = new[]
+            {
+                item.Uln < 1000000000 || item.Uln > 9999999999 ? "Uln must be between 1111111111 and 9999999999": null
```

Comment:
I thought there was more to a valid ULN? At least a checksum?

Reply:
Sat says that there is a checksum. Can you find out what it is and include it please?